### DISCUSSION TITLE
layers in group character jsons

### DISCUSSION THREAD

**Discussion by fryman444** [Vortex2Oblivion/LeatherEngine-Extended-Support#8] [Ideas]
basically just a new option (that is optional) in group char jsons that sets a character to a layer
that way if you want a character to be the first character in the chart editor, but be on a different layer during gameplay, then you can set its layer
{
    "name": "shaggy",
    "positionOffset": [0,0],
    "layer": 2
},
{
    "name": "matt",
    "positionOffset": [250,0],
    "layer": 1
}

**Comment by Vortex2Oblivion** [comment]
Group characters are added top to bottom so just switch up the order in there

And `setActorLayer()` exists in lua anyways
